wiki/markdown/index.md

```diff
@@ -15,4 +15,21 @@ For supporting latest features, Eiffel REMReM Publish should use the latest vers
 ## Components
 
 *   REMReM Publish CLI (Command Line Interface)
-*   REMReM Publish Service
\ No newline at end of file
+*   REMReM Publish Service
+
+## Compatibility
+Both [`generate`](https://github.com/eiffel-community/eiffel-remrem-generate) and `publish` services use [`semantics`](https://github.com/eiffel-community/eiffel-remrem-semantics) library. Below is compatibility table of particular versions.
+| `semantics` | `generate`          | `publish`           |
+|-------------|---------------------|---------------------|
+| `2.0.3`     | `2.0.2`             | `2.0.0`             |
+| `2.0.4`     | `2.0.3`             | `2.0.1`             |
+| `2.0.5`     | `2.0.4`             | `2.0.2` - `2.0.5`   |
+| `2.0.6`     | `2.0.5` - `2.0.9`   | `2.0.6` - `2.0.9`   |
+| `2.0.7`     |                     | `2.0.10`            |
+| `2.0.8`     | `2.0.10`            | `2.0.11`            |
+| `2.0.9`     | `2.0.11`            | `2.0.12`            |
+| `2.0.11`    |                     | `2.0.13`            |
+| `2.0.12`    | `2.0.12` - `2.0.13` | `2.0.14` - `2.0.15` |
+| `2.0.13`    | `2.0.14` - `2.0.17` | `2.0.16` - `2.0.22` |
+| `2.1.0`     | `2.1.0` - `2.1.2`   |                     |
+| `2.2.1`     | `2.1.3` - `2.1.4`   | `2.0.23`            |
```

Comment:
This doesn't look like something that belongs in this PR. Should it have been included in the PR(s) that document the Generate/Publish compatibility (or did I imagine that there is such a beat)?

Reply:
Oh, really, I must have mangled something...